src/main/java/net/masterthought/cucumber/ReportBuilder.java

```diff
@@ -264,4 +194,111 @@ private void generateErrorPage(Exception exception) throws IOException {
         errorPage.generatePage();
         System.out.println(exception);
             }
+
+    public static final class Builder {
```

Comment:
If it make sense to keep java doc on Builder class (which helps in creating ReportBuilder instance then I would add java doc for those builder methods